packages/core/src/components/html-table/_html-table.scss

```diff
@@ -35,6 +35,7 @@ Markup:
 </table>
 
 .#{$ns}-small - Small, condensed appearance
+.#{$ns}-condensed - Small, condensed appearance for just this table and tabular elements
```

Comment:
alphabetize please. then `small` could note that it affects child components, instead of here.

Reply:
fixed